### ISSUE TITLE
Could not resolve dependencies android:android:jar:4.4.2_r4

### ISSUE THREAD

**Issue by bagage** [haiwen/seadroid#268]
Simply following the `README` to generate an APK with maven 3.1.1 as stated, I run into the given issue when running `mvn clean package`:

> [ERROR] Failed to execute goal on project seadroid: Could not resolve dependencies for project com.seafile.seadroid2:seadroid:apk:1.3.0: Failure to find android:android:jar:4.4.2_r4 in https://github.com/haiwen/maven-repo/raw/master was cached in the local repository, resolution will not be reattempted until the update interval of custom maven repository has elapsed or updates are forced -> [Help 1]
> [ERROR] 
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR] 
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

I am new to `maven` build system, so I most probably must have done something wrong. But it would be great that the README is descriptive enough to successfully generate an APK for new commers easing contributions?

**Comment by lins05** [comment]
Yeah, the building with maven section of the README is outdated. For a
quick fix you can refer to our travis build script
https://github.com/haiwen/seadroid/tree/master/.travis.yml#L9 . We'll
update the README later.

**Comment by bagage** [comment]
OK following the travis file did work. Even if the README is misleading, this issue could be closed now @lins05 .

**Comment by lins05** [comment]
@bagage Can you summarize how you make it work and send us a pull request to update the REAME? :)
